posts/2023-09-13-Primeur.adoc

```diff
@@ -0,0 +1,52 @@
+
+
+== Why choose Open Liberty?
+
+Primeur says they fell in love with Open Liberty when using it within this project. The two greatest features that Primeur values from Liberty are its invisibility and observability. Although these features may seem like a contradiction and might not be something you have considered when selecting a cloud-native runtime, these two factors are critical for Primeur’s business. This is because the perspective of an ISV embedding Open Liberty inside its products is different from that of a business customer installing Open Liberty (whether on-prem or in a cloud environment) to run its business application.
+
+
+== Invisibility
+
+The great benefit of Open Liberty’s ability to remain invisible to their clients, is that Primeur can embed, completely configure, and operate Open Liberty within their DATA ONE product without their customers having to be aware that Open Liberty is even there. Their customers need only care about DATA ONE. So customers’ developers don’t have to skill up on yet another technology or care about how to operate the runtime that their platform is based upon.
+
+
+== Observability
+
+In almost complete contrast to invisibility, Primeur also required a runtime that enables great observability. As an ISV, Primeur are aware that shipping products, installing them at customer sites in the most frictionless way, and adopting them for production environments is just the beginning of what can be a long journey that can last several years until the next major product version upgrade. During this time, customers raise support requests to investigate problems that arise in the product but whose root cause can often be challenging to identify.
+
+In the ever-increasing complexity of application architecture, cloud-native tooling and distributed cloud infrastructure, bottlenecks and failure points can be very remote and well-hidden making them especially difficult to detect. It is vitally important for Primeur to be able to diagnose any problems the occur as quickly as possible to limit the impact on their customer’s business. This is why observability is critical for Primeur. Their support team needs to be able to effectively and efficiently monitor, trace and generally probe Open Liberty until they are able to identify the root cause of the problem that the customer is facing. Open Liberty offers a wide range of problem-determination tools and techniques to enable Primeur to achieve this. Moreover, Primeur's combined use of Open Liberty and link:https://developer.ibm.com/blogs/introducing-the-ibm-semeru-runtimes/[IBM Semeru Runtimes JDK] enables them to also diagnose problems at the JVM level in a much more efficient and easier way too.
```

Comment:
```suggestion

```

Reply:
(ie remove this para and just have a single condensed para in the Observability section)